plugins.json

```diff
@@ -580,5 +580,15 @@
 		"version": "1.0.1",
       		"min_version": "4.2.0",
 		"variant": "both"
+	},
+	"resourcepack-packager": {
```

Comment:
Please use snake_case with underscores for the ID (also applies to the file name).